dependency.lic

```diff
@@ -1170,19 +1169,12 @@ def toggle_developer_mode
 end
 
 def use_lich_fork
-  echo 'Now switching onto the DR lich fork...'
-  start_script('repository', ['unset-lich-updatable'])
+  echo 'Checking for updates of the DR lich fork...'
   $manager.queue_lich
 end
 
 def use_lich_main
-  echo 'Now switching onto the lich mainline...'
-  start_script('repository', ['set-lich-updatable'])
-  pause
-  start_script('repository', ['download-lich'])
-  pause
-  pause 1 while Script.running?('repository')
-  echo('New version of lich installed, please restart Lich.')
+  echo 'Deprecated setting, only the fork is supported...'
```

Comment:
❓ This removes the ability to switch off of the fork?

Reply:
yeah, do we want to support that? As it is here it will just update to the fork again the next restart, I thought that was what we wanted?

Reply:
My thought was that we'd make the fork the default and they have to manually move back to the base lich.

I haven't given it a lot of thought and don't have a strong preference right now.